SPECS/initramfs/initramfs.spec

```diff
@@ -101,6 +102,7 @@ elif [ -d %{_localstatedir}/lib/rpm-state/initramfs/pending ]; then \
         fi \
     done; \
 fi \
+%grub2_post
```

Comment:
Do we need a similar entry for `%posttrans`? I don't 100% recall when each flow runs, but maybe better to be defensive and do both.

Reply:
Yes, I think that is a good idea. Added.